airflow/models/baseoperator.py

```diff
@@ -1691,6 +1693,16 @@ def inherits_from_empty_operator(self):
         # of its subclasses (which don't inherit from anything but BaseOperator).
         return getattr(self, "_is_empty", False)
 
+    @property
+    def start_trigger(self) -> BaseTrigger | None:
+        """Trigger when deferring task."""
+        return getattr(self, "_start_trigger", None)
```

Comment:
If we always set the value in `__init__`, why do we need to use `getattr` here?

Reply:
Yep, this is no longer needed after the initialization has been moved to `__init__`. I'll update it. Thanks!